csharp/ql/integration-tests/posix-only/inherit-env-vars/build.sh

```diff
@@ -0,0 +1,2 @@
+cp $PROJECT_TO_BUILD temp.csproj
```

Comment:
So you are testing that the build script invoked by the autobuilder can use system environment variables?

Reply:
Yes.